app/_src/gateway/kong-enterprise/cp-outage-handling.md

```diff
@@ -0,0 +1,142 @@
+
+In this setup you will need to designate one backup node. The backup node must have read/write access to the S3 compatible storage volume. This node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the storage volume. Nodes are initialized with fallback configs via environment variables, including `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, and `AWS_DEFAULT_REGION`. The backup node must have read/write access to the storage volume. This node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the storage volume. A backup node should not be used to proxy traffic. A single backup node is sufficient for all deployments. For more information about the data that is set in the environment variables review the [AWS environment variable configuration documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-envvars.html).
+
+Using Docker Compose you can configure the backup data plane like this:
+
+```yaml
+kong-exporter:
+    image: 'kong/kong-gateway:latest'
+    ports:
+      - '8000:8000'
+      - '8443:8443'
+    environment:
+      <<: *other-kong-envs
+      AWS_REGION: 'us-east-2'
+      AWS_ACCESS_KEY_ID: <access_key_write>
+      AWS_SECRET_ACCESS_KEY: <secret_access_key_write>
+      KONG_CLUSTER_FALLBACK_CONFIG_STORAGE: s3://test-bucket/test-prefix
+      KONG_CLUSTER_FALLBACK_CONFIG_EXPORT: "on"
+
+```
+
+This node is responsible for writing to the S3 bucket when it receives a new configuration. If the node version is `3.2.0.0`, the key name should be `test-prefix/3.2.0.0/config.json`.
+Both the control plane and data plane can be configured to export configurations.
+
+A new data plane can be configured to load a configuration from a bucket if the control plane is not reachable using the following environment variables: 
+
+```yaml
+kong-dp-importer:
+    image: 'kong/kong-gateway:latest'
+    ports:
+      - '8000:8000'
+      - '8443:8443'
+    environment:
+      <<: *other-kong-envs
+      AWS_REGION: 'us-east-2'
+      AWS_ACCESS_KEY_ID: <access_key_read>
+      AWS_SECRET_ACCESS_KEY: <secret_access_key_read>
+      KONG_CLUSTER_FALLBACK_CONFIG_STORAGE: s3://test-bucket/test-prefix
+      KONG_CLUSTER_FALLBACK_CONFIG_IMPORT: "on"
+
+```
+
+
+
+{% endnavtab %}
+{% navtab GCP Cloud Storage %}
+## Prerequisites
+
+* A GCP cloud storage bucket
+* Read/write credentials for the bucket.
+
+
+## Configuration
+
+In this setup you will need to designate one backup node. The backup node must have read/write access to the storage volume. This node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the storage volume. The backup node will need **write** access to the storage volume. A backup node should not be used to proxy traffic. A single backup node is sufficient for all deployments.
```

Comment:
Same. "The backup node will need **write** access ..." is duplicated with "The backup node must have read/write access to ...".
```suggestion
In this setup you will need to designate one backup node. The backup node must have read/write access to the storage volume, and the data plane nodes supposed to be provisioned must have read access to the storage volume. This node is responsible for communicating the state of the {{site.base_gateway}} `kong.conf` configuration file from the control plane to the storage volume. A backup node should not be used to proxy traffic. A single backup node is sufficient for all deployments.
```